kafka_consumer/check.py

```diff
@@ -187,13 +187,13 @@ def check(self, instance):
                 continue
             consumer_lag = highwater_offsets[(topic, partition)] - consumer_offset
             if consumer_lag < 0:
-                # This is a really bad scenario because new messages produced to
-                # the topic are never consumed by that particular consumer
-                # group. So still report the negative lag as a way of increasing
-                # visibility of the error.
-                title = "Consumer lag for consumer negative."
-                message = "Consumer lag for consumer group: {group}, topic: {topic}, " \
-                    "partition: {partition} is negative. This should never happen.".format(
+                # this will result in data loss, so emit an event for max visibility
+                title = "Negative consumer lag for group: {group}.".format(group=consumer_group)
+                message = "Consumer lag for Kafka consumer group: {group}, topic: {topic}, " \
+                    "partition: {partition} is negative.\n\n" \
+                    "This is theoretically impossible and will result in data loss " \
```

Comment:
This looks better, thanks a lot for making it clearer. Should we tweak this to: _this is theoretically impossible, yet may occur, and will will result in data loss_ ?